### ISSUE TITLE
Create a loss class for MSE and cross entropy loss

### ISSUE THREAD

**Issue by SIDD017** [SIDD017/PyTorch-Framework#5]
Implement a forward and backward method for each
